qmlnotifyservice.h

```diff
@@ -9,18 +9,19 @@ class QQmlApplicationEngine;
 /**
  * Simple notify service for qml.
  * Use :
+ * @code cpp 
  * #include <qmlnotifyservice.h>
- *
  * QmlNotificationService::init();
  * auto service = QmlNotificationService::NotificationService::getService()
  * service->setNotify("title", "text", "UrlOfImage", NotificationData::Normal)
- *
+ * @endcode 
  *
  * in qml :
- *
+ * @code cpp 
```

Comment:
```suggestion
 * @code qml 
```
This is not c++ code